src/targets/docker.ts

```diff
@@ -0,0 +1,121 @@
+import { logger as loggerRaw } from '../logger';
+import { TargetConfig } from '../schemas/project_config';
+import { BaseArtifactProvider } from '../artifact_providers/base';
+import { ConfigurationError } from '../utils/errors';
+import { checkExecutableIsPresent, spawnProcess } from '../utils/system';
+import { BaseTarget } from './base';
+
+const logger = loggerRaw.withScope('[docker]');
+
+const DEFAULT_DOCKER_BIN = 'docker';
+
+/**
+ * Command to launch docker
+ */
+const DOCKER_BIN = process.env.DOCKER_BIN || DEFAULT_DOCKER_BIN;
+
+/** Options for "docker" target */
+export interface DockerTargetOptions extends TargetConfig {
+  username: string;
+  password: string;
+  /** Source image path, like `us.gcr.io/sentryio/carft` */
```

Comment:
```suggestion
  /** Source image path, like `us.gcr.io/sentryio/craft` */
```
🙈

Reply:
Random spam: Let's rename this repo to carft. I really like the sound of it. So quirky :)